bundles/statistics/statsgrid2016/service/ClassificationService.js

```diff
@@ -200,10 +200,10 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.ClassificationService',
             if(!ranges.max) {
                 ranges.max = 4;
             }
-            var x = d3.scaleSqrt()
+            var x = d3.scaleLinear()
```

Comment:
We should continue using `scaleSqrt()` because the values should be mapped to the area of the circles. See: [Interactive Data Visualization for the Web](https://books.google.fi/books?id=NmwvDwAAQBAJ&pg=SA6-PA52&lpg=SA6-PA52&dq=visualization+area+of+circle+scalesqrt&source=bl&ots=D2WSYvvxHk&sig=Mu3RcVs4A9J6C6Gn22-UUqxKR-E&hl=en&sa=X&ved=0ahUKEwj2hMfhyMPZAhVSJlAKHeV9Ac8Q6AEIOTAC)

If we use `scaleLinear()` we are mapping the values to radius of the circle, which is perceptually wrong.

Reply:
The task was to implement scaling similar to the example 5.cg.5 in https://www.e-education.psu.edu/geog486/node/1869.  Mapping the values to area is generally a good idea, but here we are just presenting the classifications instead of visualizing the actual values on the map.

Reply:
Ah, was this the reason the circles were different sizes in the legend and on the map? If so, we should use `scaleSqrt()` in both map and legend.